### DISCUSSION TITLE
how to install phonony and phonolammps packages based on deepkit/lmp

### DISCUSSION THREAD

**Discussion by liyibaipnu** [deepmodeling/deepmd-kit#711] [Q&A]
Dear DeepKit Users

I am doing some dpmd tests using deepkit-1.1.4 . To calculate the phonon spetrum, I need install phonony and phonolammps packages. I found a tutorial about how to install phonony and phonolammps packages on Independently installed lammps（Not lmp in deepkit）https://zhuanlan.zhihu.com/p/344803376.
I would be appreciate if anyone can give any suggestion on how to install phonony and phonolammps packages based on deepkit/lmp.

Thank your very much

Best Regards

Liyi

**Comment by njzjz** [comment]
It should be available in the version above v1.2.0. If you still need to compile with v1.1.4, see https://github.com/deepmodeling/deepmd-kit/discussions/667#discussioncomment-786282
